M2/Macaulay2/packages/VectorGraphics/VectorGraphics.js

```diff
@@ -461,47 +468,10 @@ class Matrix extends Float32Array {
 
     inverse() // size 4 only!
     {
-	var A2323 = this[10] * this[15] - this[11] * this[14],
-	    A1323 = this[9] * this[15] - this[11] * this[13],
-	    A1223 = this[9] * this[14] - this[10] * this[13],
-	    A0323 = this[8] * this[15] - this[11] * this[12],
-	    A0223 = this[8] * this[14] - this[10] * this[12],
-	    A0123 = this[8] * this[13] - this[9] * this[12],
-	    A2313 = this[6] * this[15] - this[7] * this[14],
-	    A1313 = this[5] * this[15] - this[7] * this[13],
-	    A1213 = this[5] * this[14] - this[6] * this[13],
-	    A2312 = this[6] * this[11] - this[7] * this[10],
-	    A1312 = this[5] * this[11] - this[7] * this[9],
-	    A1212 = this[5] * this[10] - this[6] * this[9],
-	    A0313 = this[4] * this[15] - this[7] * this[12],
-	    A0213 = this[4] * this[14] - this[6] * this[12],
-	    A0312 = this[4] * this[11] - this[7] * this[8],
-	    A0212 = this[4] * this[10] - this[6] * this[8],
-	    A0113 = this[4] * this[13] - this[5] * this[12],
-	    A0112 = this[4] * this[9] - this[5] * this[8];
-	var det =
-	    this[0] * ( this[5] * A2323 - this[6] * A1323 + this[7] * A1223 )
-	    - this[1] * ( this[4] * A2323 - this[6] * A0323 + this[7] * A0223 )
-	    + this[2] * ( this[4] * A1323 - this[5] * A0323 + this[7] * A0123 )
-	    - this[3] * ( this[4] * A1223 - this[5] * A0223 + this[6] * A0123 );
+	var A2323 = this[10] * this[15] - this[11] * this[14], A1323 = this[9] * this[15] - this[11] * this[13], A1223 = this[9] * this[14] - this[10] * this[13], A0323 = this[8] * this[15] - this[11] * this[12], A0223 = this[8] * this[14] - this[10] * this[12], A0123 = this[8] * this[13] - this[9] * this[12], A2313 = this[6] * this[15] - this[7] * this[14], A1313 = this[5] * this[15] - this[7] * this[13], A1213 = this[5] * this[14] - this[6] * this[13], A2312 = this[6] * this[11] - this[7] * this[10], A1312 = this[5] * this[11] - this[7] * this[9], A1212 = this[5] * this[10] - this[6] * this[9], A0313 = this[4] * this[15] - this[7] * this[12], A0213 = this[4] * this[14] - this[6] * this[12], A0312 = this[4] * this[11] - this[7] * this[8], A0212 = this[4] * this[10] - this[6] * this[8], A0113 = this[4] * this[13] - this[5] * this[12], A0112 = this[4] * this[9] - this[5] * this[8];
```

Comment:
I believe @d-torrance broke these lines up to fix a Debian package linting issue. See #1646

Reply:
that's annoying. that's not even M2 code, so why would anyone care how long the line is?

Reply:
(this particular code was generated automatically BTW)

Reply:
Debian's policy is very particular about including the full source code for all of the software in its archive.  They find Javascript libraries in particular worrisome, as often only minified source is included, but they don't consider that actual source code.  To try to weed minified Javascript out of the archive, the lintian utility throws some warnings and an error if it finds a Javascript file with a line longer than 256 characters in a Debian package.

Reply:
OK no problems, fixed.